src/AuthService/Application/External/Infrastructure/Configurations/UserConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Configurations;
+
+public class UserConfiguration : IEntityTypeConfiguration<User>
+{
+    public void Configure(EntityTypeBuilder<User> builder)
+    {
+        builder.ToTable("Users");
```

Comment:
Why have you decided to not have Users DbCSet in AuthDbContext but to have this hardcoded string "Users" here?